groundstation/static/js/components/Help.jsx

```diff
@@ -366,6 +366,11 @@ const Help = () => {
                   {command.about_info != null ? (
                     <p style={{ paddingLeft: '5%' }}>{command.about_info}</p>
                   ) : null}
+                  {command.arg_labels != null ? (
```

Comment:
Can we move the argument list to replace the 'number of arguments' field next to the command name (so it looks like what you would type in).

eg. housekeeping.get_hk(Limit, Before_id, Before_time) instead of housekeeping.get_hk (3)

We can then have this format for return values instead.